spec/index.bs

```diff
@@ -1022,19 +1022,21 @@ This algorithm will return a string that can be used with an
 It will ensure that the Trusted Type [=enforcement=] rules were respected.
 
 Given a {{TrustedType}} type (|expectedType|), a [=realm/global object=] (|global|),
-{{TrustedType}} or a string (|input|), a string (|sink|) and a string (|sinkGroup|), run these steps:
+an [=ECMAScript language value=] (|input|), a string (|sink|) and a string (|sinkGroup|), run these steps:
```

Comment:
If this now takes an ECMAScript language value which is what it's passed from the validate algorithm which gets it from WebIDL, does that "just work" for all the other uses of this algorithm? Or does it as I suspect break those call sites?